### PR TITLE
print generated encryption key passwords directly after generating the keys…

### PR DESCRIPTION
**Pull request by weizenspreu**
…so that errors during the encryption do not lead to a potentially massive data loss

### CODE REVIEW

**Comment by MorrisJobke** [comment]
cc @nextcloud/designers

**Review by nickvergessen — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]

**Review by LukasReschke — APPROVED** [review]

**Review by LukasReschke — CHANGES_REQUESTED** [review]
```
1) OCA\Encryption\Tests\Crypto\EncryptAllTest::testEncryptAll
922s
121
Expectation failed for method name is equal to <string:encryptAllUsersFiles> when invoked at sequence index 1.
922s
122
Mocked method does not exist.
922s
123
```

Tests failing :)

File: apps/encryption/tests/Crypto/EncryptAllTest.php [hunk 1]
```diff
@@ -149,8 +149,8 @@ public function testEncryptAll() {
 
 		$this->util->expects($this->any())->method('isMasterKeyEnabled')->willReturn(false);
 		$encryptAll->expects($this->at(0))->method('createKeyPairs')->with();
-		$encryptAll->expects($this->at(1))->method('encryptAllUsersFiles')->with();
-		$encryptAll->expects($this->at(2))->method('outputPasswords')->with();
+                $encryptAll->expects($this->at(1))->method('outputPasswords')->with();
```
Comment (nickvergessen) [inline apps/encryption/tests/Crypto/EncryptAllTest.php]: whitespace breakage

**Comment by MorrisJobke** [comment]
Could I ask you to sign-off the commits: For details see https://github.com/nextcloud/server/blob/master/CONTRIBUTING.md#sign-your-work

Thanks

**Comment by MorrisJobke** [comment]
It would also be nice to have only one commit.(squash them into one and then add the sign-off message there)

Regarding the CI timeouts: they are fixed on current master - so a rebase to the current master would be nice as well. Sorry for this mess :/

**Comment by weizenspreu** [comment]
Proceeds here... https://github.com/nextcloud/server/pull/6448

**Pull request closed without merging**